R/ReactantExtra/build_tarballs.jl

```diff
@@ -0,0 +1,222 @@
+BAZEL_BUILD_FLAGS+=(--define=llvm_enable_zlib=false)
+BAZEL_BUILD_FLAGS+=(--verbose_failures)
+
+if [[ "${bb_full_target}" == *darwin* ]]; then
+    BAZEL_BUILD_FLAGS+=(--define=build_with_mkl=false --define=enable_mkl=false)
+    BAZEL_BUILD_FLAGS+=(--define=gcc_linux_x86_32_1=false)
+    BAZEL_BUILD_FLAGS+=(--define=gcc_linux_x86_64_1=false)
+    BAZEL_BUILD_FLAGS+=(--define=gcc_linux_x86_64_2=false)
+    BAZEL_BUILD_FLAGS+=(--define=gcc_linux_aarch64=false)
+    BAZEL_BUILD_FLAGS+=(--define=clang_macos_x86_64=true)
+    BAZEL_BUILD_FLAGS+=(--macos_minimum_os=10.14)
+    export MACOSX_DEPLOYMENT_TARGET=10.14
+    BAZEL_BUILD_FLAGS+=(--action_env=MACOSX_DEPLOYMENT_TARGET=10.14)
+    BAZEL_BUILD_FLAGS+=(--host_action_env=MACOSX_DEPLOYMENT_TARGET=10.14)
+    BAZEL_BUILD_FLAGS+=(--repo_env=MACOSX_DEPLOYMENT_TARGET=10.14)
+    BAZEL_BUILD_FLAGS+=(--test_env=MACOSX_DEPLOYMENT_TARGET=10.14)
+    BAZEL_BUILD_FLAGS+=(--noincompatible_remove_legacy_whole_archive)
+    BAZEL_BUILD_FLAGS+=(-s)
+    env
+fi
+
+if [[ "${bb_full_target}" == *freebsd* ]]; then
+    BAZEL_BUILD_FLAGS+=(--define=gcc_linux_x86_32_1=false)
+    BAZEL_BUILD_FLAGS+=(--define=gcc_linux_x86_64_1=false)
+    BAZEL_BUILD_FLAGS+=(--define=gcc_linux_x86_64_2=false)
+    BAZEL_BUILD_FLAGS+=(--define=gcc_linux_aarch64=false)
+    BAZEL_BUILD_FLAGS+=(--define=freebsd=true)
+fi
+
+if [[ "${bb_full_target}" == *i686* ]]; then
+    BAZEL_BUILD_FLAGS+=(--define=build_with_mkl=false --define=enable_mkl=false)
+fi
+
+# julia --project=. -e "using Pkg; Pkg.instantiate(); Pkg.add(url=\"https://github.com/JuliaInterop/Clang.jl\", rev=\"vc/cxx_parse2\")"
+BAZEL_BUILD_FLAGS+=(--action_env=JULIA=julia)
+bazel ${BAZEL_FLAGS[@]} build ${BAZEL_BUILD_FLAGS[@]} ...
+rm bazel-bin/libReactantExtraLib*
+mkdir -p ${prefix}/lib
+cp bazel-bin/libReactantExtra* ${prefix}/lib
+cp bazel-bin/*.jl ${prefix}
+"""
+
+augment_platform_block = ""
+
+# determine exactly which tarballs we should build
+builds = []
+    
+# Dependencies that must be installed before this package can be built
+
+dependencies = []
+
+# The products that we will ensure are always built
+products = Product[
+    LibraryProduct(["libReactantExtra", "libReactantExtra"],
+                   :libReactantExtra, dont_dlopen=true),
+    FileProduct("Affine.inc.jl", Symbol("Affine.inc.jl")),
+    FileProduct("Arith.inc.jl", Symbol("Arith.inc.jl")),
+    FileProduct("Builtin.inc.jl", Symbol("Builtin.inc.jl")),
+    FileProduct("Enzyme.inc.jl", Symbol("Enzyme.inc.jl")),
+    FileProduct("Func.inc.jl", Symbol("Func.inc.jl")),
+    FileProduct("StableHLO.inc.jl", Symbol("StableHLO.inc.jl")),
+    FileProduct("CHLO.inc.jl", Symbol("CHLO.inc.jl")),
+    FileProduct("VHLO.inc.jl", Symbol("VHLO.inc.jl")),
+    # FileProduct("libMLIR_h.jl", Symbol("libMLIR_h.jl")),
+]
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = expand_cxxstring_abis(supported_platforms(; experimental=true))
```

Comment:
`experimental` is no-op nowadays, it's kept only for compatibility reasons
```suggestion
platforms = expand_cxxstring_abis(supported_platforms())
```